src/main/java/seedu/address/model/person/contact/Phone.java

```diff
@@ -35,19 +38,19 @@ public static boolean isValidPhone(String test) {
 
     @Override
     public String toString() {
-        return value;
+        return getValue();
     }
 
     @Override
     public boolean equals(Object other) {
         return other == this // short circuit if same object
-                || (other instanceof Phone // instanceof handles nulls
-                && value.equals(((Phone) other).value)); // state check
+            || (other instanceof Phone // instanceof handles nulls
+            && getValue().equals(((Phone) other).getValue())); // state check
     }
 
     @Override
     public int hashCode() {
-        return value.hashCode();
+        return getValue().hashCode();
     }
```

Comment:
We can put these Logic to `Contact` class so we don't need to repeat every time.